server-phoenix/test/helios_web/controllers/events_api/pub.pem

```diff
@@ -0,0 +1,9 @@
+-----BEGIN PUBLIC KEY-----
+MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA3ETTMJbnAeVPi+BIqogG
+hfN0CueTVGVdLF6idoNlzEoPjY0L2uqpJX4dU8B4fTf1HGMsyU8AMPSdnnMpoTvY
+ZtQxQxkVcOFwfdQjQ648RAXqIm37F5WCgchsai/L9sN1Kg8RRvC0r0Ol8B4dLGoj
+IRdx1GFLwlXa3DqkMvCSRh5zintO7vBelgCOwhyOfpbui9iXCW4mVWiP2xTB1+Gs
+pMyaCfZriRkPb6/8RMK8PIcIW878ODUUuRSZH+guxjs8c2DL7tFTvkMGczkwXXi3
+92X6GRxWT0jNGMIUPdtWmgEUXjOBuYhuLyjHz1RqPiRtDMYfJNpu7yI1VtA2Gp1r
+xQIDAQAB
+-----END PUBLIC KEY-----
```

Comment:
Lets add the private key as well in case we need to recreate any of the above payloads (assuming these are both created just for this test). If this is your actual public key, lets switch it to one we create just for testing.

Reply:
added